shaders/glsl/animations.glsl

```diff
@@ -0,0 +1,11 @@
+#define WIND_SPEED 8.0
```

Comment:
@MegadronA03  are u sure this file in a correct directory? I think it should be in /glsl/options/preset